src/locales/en.json

```diff
@@ -203,21 +203,21 @@
   },
   "about": {
     "intro_header": "What is the microCOVID project?",
-    "basic_description": "This is a project to quantitatively estimate the COVID risk to you from your ordinary daily activities. We trawled the scientific literature for data about the likelihood of getting COVID from different situations, and combined the data into a model that people can use. We estimate COVID risk in units of microCOVIDs, where 1 microCOVID = a one-in-a-million chance of getting COVID.",
+    "basic_description": "A calculator of your COVID-19 risk from ordinary daily activities, estimated in microCOVID units (1 mC being a one-in-a-million chance of contracting COVID-19, based on the scientific literature on it.",
```

Comment:
Again, help me understand what is better about this?

Reply:
It is super long, which makes figuring it all out prior to translating it a pain.
Making it longer still, the amount of entropy and complex language is a tall order before or after translation.
Introducing errors in translation is likely, or the hardest strings to decipher are left untranslated, helping nobody.